### PR TITLE
Query Store Best practices - update the default Max Storage Size value

### PR DESCRIPTION
**Pull request by relsna**
Tiny change, but the Max Storage size is now 1000 MB by default (since 2019).

### CODE REVIEW

**Comment by PRMerger10** [comment]
@relsna : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
@WilliamDAssafMSFT - Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
#assign:@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger20** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: microsoftdocs/public-repo-pr-review-team

File: docs/relational-databases/performance/best-practice-with-the-query-store.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ To adjust these options as your workload grows, see [Keep Query Store adjusted t
 
 | Configuration | Description | Default | Comment |
 | --- | --- | --- | --- |
-| MAX_STORAGE_SIZE_MB |Specifies the limit for the data space that Query Store can take inside the customer database |100 |Enforced for new databases |
+| MAX_STORAGE_SIZE_MB |Specifies the limit for the data space that Query Store can take inside the customer database |1000 |Enforced for new databases |
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/performance/best-practice-with-the-query-store.md:49]:
```suggestion
| MAX_STORAGE_SIZE_MB |Specifies the limit for the data space that Query Store can take inside the customer database |100 prior to SQL Server 2019<BR>1024 starting with SQL Server 2019 |Enforced for new databases |
```

**Comment by PRMerger16** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by WilliamDAssafMSFT** [comment]
Thanks @relsna for taking the time to submit this PR! Nice catch, this did need to be updated. I added a commit on top. Your commit will be merged and this article update is in flight. After the updated article is published, GitHub will list you as an article contributor. 

Thanks again for this feedback, your suggestions help improve our documentation.

#sign-off

**Pull request merged**